dnd-character-generator/src/components/AttacksSpells.jsx

```diff
@@ -1,7 +1,69 @@
-export function AttacksSpells() {
-    return (
-      <div className="p-2 border border-black bg-[#e1d3b8] text-center font-bold">
-        AttacksSpells
+import { useState, useEffect } from 'react';
+
+export function AttacksSpells({ spells = [], attacks = [] }) {
+  const [customSpells, setCustomSpells] = useState(spells);
+  const [newSpell, setNewSpell] = useState('');
+
+  // Update spells als character opnieuw gegenereerd wordt
+  useEffect(() => {
+    setCustomSpells(spells);
+  }, [spells]);
+
+  const addSpell = () => {
+    const cleaned = newSpell.trim();
+    if (cleaned !== '' && !customSpells.includes(cleaned)) {
+      setCustomSpells([...customSpells, cleaned]);
+      setNewSpell('');
+    }
+  };
+
+  return (
+    <div className="p-2 border border-black bg-[#e1d3b8]  text-sm">
+      <h2 className="text-center mb-2 font-bold">Attacks & Spellcasting</h2>
+
+      {/* Attacks */}
+      <div className="mb-4">
+        <p className="underline">Attacks:</p>
```

Comment:
`<p>` is eigenlijk voor lopende teksten. Ik zou hier gewoon een spannetje van maken

```suggestion
        <span className="underline">Attacks:</span>
```